Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => {
+    return (
+      <div>
+        <img src={'https://ibin.co/3wnC6SgIOJud.png'} />
+        
+      </div>
+    );
```

Comment:
social card looks good. Just need to use CSS to get the display correct. Well done!